### ISSUE TITLE
Add checks for Python (and other executables?)

### ISSUE THREAD

**Issue by jprestop** [dtcenter/METplus#26]
Add a check for Python 2.7.X and exit if lower.

Do we also want to consider adding checks for the other required apps?

- METplus requires the following to be installed on your system:
  - ncdump utility
    - http://www.unidata.ucar.edu/downloads/netcdf/index.jsp
  - ncap2 utility
    - http://nco.sourceforge.net/
  - convert utility (part of ImageMagick)
    - https://www.imagemagick.org/script/binary-releases.php
  - wgrib2 utility
    - http://www.cpc.noaa.gov/products/wesley/wgrib2/compile_questions.html
  - egrep utility
    - http://directory.fsf.org/wiki/Grep
  - rm, cut, tr utilities (standard on Linux)

**Comment by georgemccabe** [comment]
Check for Python version is already done in master_metplus.py. The script will error and exit if the user's version is less than the minimum required version of Python. I moved this check to metplus_check_python_version.py so that it can be imported by each wrapper that can be called stand alone. 

However, if f-strings are used in a wrapper, a SyntaxError will be shown before the error message can be reported. Some wrappers don't use f-strings and some use them minimally so they could be modified to not use them for the sake of this error message. Some wrappers use them extensively. The group decided that we want to utilize f-strings despite issues with old versions. To get around this, we could add an f-string message about the version at the top of the wrappers so that if an old version of Python is used, the SyntaxError that is reported will mention version issues to hint to the user that the python version is the problem.

METplus also already handles checking if the external exes exist if they need to be used. The config.getexe function makes sure the exe exists when it is requested.

In short, this issue can be closed unless we want to make changes to the wrappers that use f-strings so that running them stand alone will report the error correctly. Most uses of METplus will be run through master_metplus.py so this is not an issue for those cases.
